contents/blog/posthog-vs-ga4.md

```diff
@@ -0,0 +1,536 @@
+---
+date: 2023-10-12
+title: "In-depth: PostHog vs Google Analytics 4"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/posthog-vs-ga4/posthog-vs-ga4.jpeg
+featuredImageType: full
+author:
+    - lior-neu-ner
+category: General
+tags:
+    - Comparisons
+    - Guides
+---
+
+Want to understand how PostHog and Google Analytics 4 (GA4) are different? Here's the short answer:
+
+- **Google Analytics** is designed for marketers to understand user traffic, performance of different marketing channels, and return on ad spend.
+
+- **PostHog** is an all-in-one platform built to help engineers create better products. It includes analytics and a whole bunch more – like session replays, feature flags, A/B testing, and surveys.
+
+In this article we’ll explore these differences in more detail, and take a deep dive into the analytics features of both.
+
+## How is PostHog different to Google Analytics?
+
+### 1. PostHog does more than analytics
+
+[Analytics](/product-analytics) is a core part of our platform, but it's just one component in a set of important tools for product teams. PostHog enables you to collect feedback with [surveys](/docs/surveys), understand user behavior with [session replays](/session-replay), test changes with [A/B tests](/ab-testing), and deploy changes with [feature flags](/feature-flags).
+
+### 2. PostHog is built for engineers
+
+Anyone can use PostHog – just [ask our marketing team](/blog/posthog-marketing) – but PostHog is designed to meet the needs of engineers. Our goal is to help developers [be better at product](/blog/helping-engineers-to-product).
+
+### 3. PostHog is open source and transparent
+
+PostHog is built with transparency at its core. We work in the open and give full access to our [source code](https://github.com/PostHog), and enable you to [build integrations](/docs/apps/build) and [other services](/blog/how-we-built-an-app-server) on top of the product. You can even give feedback on [our public roadmap](/roadmap).
+
+<ArrayCTA /> 
+
+## PostHog and Google Analytics feature comparison
```

Comment:
1. Autocapture? I've heard complaints about how long it takes to implement GA (and how expensive it is). Especially GA4 makes this complicated. 
2. Mobile analytics. Both have it, but focus on different areas. Potentially worth noting.

Reply:
On autocapture, looks like GA4 has it under "enhanced analytics": https://support.google.com/analytics/answer/9216061

Reply:
> Autocapture? I've heard complaints about how long it takes to implement GA (and how expensive it is). Especially GA4 makes this complicated.

Hmm this wasn't my experience with it when I set it up a few days ago. It was just pasting a few lines of code and it began auto-capturing events

> Mobile analytics. Both have it, but focus on different areas. Potentially worth noting.

Hmm mind elaborating? Seems like both PostHog and GA4 do similar things (event capture and user properties/demographics)

https://firebase.google.com/docs/analytics/get-started?technology=ios&platform=ios
https://firebase.google.com/docs/analytics/get-started?technology=android&platform=android

Reply:
Love Google's name for things. Enhanced analytics is the vaguest, most meaningless term.